src/cascade/core/db.py

```diff
@@ -0,0 +1,55 @@
+from contextlib import contextmanager
```

Comment:
We have been adding a docstring at the top of every module.

Reply:
Ok. I'll do better documentation :.(